canvas_modules/common-canvas/__tests__/toolbar/toolbar-test.js

```diff
@@ -69,6 +69,31 @@ describe("Toolbar renders correctly", () => {
 		expect(canvasToolbar.find(".toolbar-spacer")).to.have.length(0);
 	});
 
+	it("should render a Toolbar with medium size buttons", () => {
+		const toolbarConfig = {
+			leftBar: [
+				{ action: "palette", label: "Palette", enable: true },
+				{ divider: true },
+				{ action: "stop", label: "Stop Execution", enable: false },
+				{ action: "run", label: "Run Pipeline", enable: false },
+			]
+		};
+
+		const canvasToolbar = createToolbar(toolbarConfig);
+
+		// Select the toolbar buttons
+		const overflowButtons = canvasToolbar.find(".toolbar-overflow-item button");
+		const defaultButtons = canvasToolbar.find(".toolbar-item.default button");
+
+		expect(overflowButtons).to.have.length(3);
+		expect(defaultButtons).to.have.length(3);
+
+		// Verify if the buttons show up with the right size
+		expect(overflowButtons.find(".bx--btn--md")).to.have.length(3);
+		expect(defaultButtons.find(".bx--btn--md")).to.have.length(3);
+
+	});
+
```

Comment:
What about testing when size is set to "sm" -- which is what your PR is implementing.

Reply:
Add more test for sm